app/src/main/java/com/sun/livescore/data/repository/LiveEventRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.sun.livescore.data.repository
+
+import com.sun.livescore.data.model.event.EventResponse
+import com.sun.livescore.data.remote.live.LiveEventDataSource
+import com.sun.livescore.data.remote.live.LiveEventRemoteDataSource
+import io.reactivex.Single
+
+class LiveEventRepository(private val liveEventRemoteDataSource: LiveEventRemoteDataSource) :
+    LiveEventDataSource.Remote {
```

Comment:
nếu mà impl Remote thôi thì gọi là LiveEventRemoteRepository nhỉ

Reply:
mấy cái cũ em toàn gọi thế mà :(